src/strawberry_sqlalchemy_mapper/mapper.py

```diff
@@ -823,10 +825,10 @@ def convert(type_: Any) -> Any:
                 mapped_type = strawberry.interface(type_)
                 self.mapped_interfaces[type_.__name__] = mapped_type
             elif use_federation:
-                mapped_type = strawberry.federation.type(type_)
+                mapped_type = strawberry.federation.type(type_, directives=directives if directives else ())
```

Comment:
I believe this if statement here is unnecessary since the default value is already `()`. What do you think about simplifying it to `directives=directiver`, similar to what you did on line 831?

Reply:
I did that to satisfy the typings -- the federated and non-federated functions take two different parameters. One of them allows it to be None, the other does not.

Reply:
Okay I undestand.
Marked as resolved!